libraries/AP_IOMCU/iofirmware/analog.cpp

```diff
@@ -75,7 +75,7 @@ static uint16_t adc_sample_channel(uint32_t channel)
 uint16_t adc_sample_vservo(void)
 {
     const uint32_t channel = ADC_SQR3_SQ1_N(ADC_CHANNEL_IN4);
-    return adc_sample_channel(channel) * 9900 / 4096;
+    return adc_sample_channel(channel) * 3300 / 4096;
```

Comment:
if we change this then all the logging of servo voltages in POWR log message and POWER_STATUS mavlink msg will break

Reply:
Scripting wrapper then? I'm a little confused on how the correct value is getting in the logs then. If there is voltage divider going on for these pins, shouldn't that be a separate multiplier just to be descriptive?

Reply:
I would vote for:
```suggestion
    return adc_sample_channel(channel) * (3300 / 4096) * 3;
```

Or even better use the define.

```suggestion
    return adc_sample_channel(channel) * VOLTAGE_SCALING * 3;
```

Reply:
The first suggestion here won't work, because (3300/4096) is pure integer and will be a 0, which means the whole expression returns 0, the second suggestion incurs a conversion to and from a float, and lowers the accuracy of the returned number. (And if this is running on the F101 it's even worse, because there isn't a hardware FPU to do this math).